gum/backend-darwin/gumkernel-darwin.c

```diff
@@ -166,6 +167,39 @@ gum_kernel_alloc_n_pages (guint n_pages)
   return result + page_size;
 }
 
+gboolean
+gum_kernel_try_free_pages (GumAddress mem)
+{
+  mach_port_t task;
+  gsize page_size;
+  mach_vm_address_t address;
+  mach_vm_size_t *size;
```

Comment:
Missing space after `*`:

```suggestion
  mach_vm_size_t * size;
```